pkg/collector/corechecks/cluster/kubernetes_state.go

```diff
@@ -270,26 +267,26 @@ func (k *KSMCheck) processMetrics(sender aggregator.Sender, metrics map[string][
 			if !isKnownMetric(metricFamily.Name) {
 				// ignore the metric if it doesn't have a transformer
 				// or if it isn't mapped to a datadog metric name
-				log.Tracef("KSM metric '%s' is unknown for the check, ignoring it", metricFamily.Name)
+				// log.Tracef("KSM metric '%s' is unknown for the check, ignoring it", metricFamily.Name)
```

Comment:
I think you should add back the `log.Trace`

Reply:
Ok. If this trace is useful, I can add it back.
The reason why I commented it is because this trace represents 12% of the CPU time consumed by `processMetrics`, which is quite huge:
![image](https://user-images.githubusercontent.com/1437785/103623015-c6151c80-4f37-11eb-9f43-ea06dfd7fdb5.png)